yvalenta/Readme.md

```diff
@@ -22,6 +23,12 @@ the execution time.
 
 ### Sequential
 
-➜  yvalenta git:(master) ✗ ruby search.rb
+➜  yvalenta git:(master) ✗ ruby sequential_search.rb
 {"a"=>4049408, "e"=>6050373, "i"=>3550018, "o"=>4236236, "u"=>1487946}
-Run in 16.385092057 seconds.
\ No newline at end of file
+Run in 12.943612394 seconds.
+
+### Threads
+
+➜  yvalenta git:(master) ✗ ruby parallel_search.rb
```

Comment:
la parte anterior queda sobrando:

Running

To run the program just type this command in terminal:

ruby search.rb